deployability/modules/testing/tests/test_agent/test_install.py

```diff
@@ -1,83 +1,77 @@
 import grp
 import pwd
-
-from ..helpers import constants, utils, check_files
-
-
-def test_files_permissions():
-
-    missing_names = []
-    extra_names = []
-    different_names = []
-    # Get data
-    static_items, dynamic_items = check_files.get_template_items('agent')
-    current_items = check_files.get_current_items()
-    # HARDCODED: Always ignore /var/ossec/api/node_modules/
-    current_items, _ = check_files.cut_items(current_items)
-    static_items, _ = check_files.cut_items(static_items, ['/var/ossec/wodles/aws'])
-    dynamic_items, _ = check_files.cut_items(dynamic_items)
-
-
-    static_names = static_items.keys()
-    dynamic_names = dynamic_items.keys()
-    current_names = current_items.keys()
-
-
-    # Missing files/directories
-    missing_names = set(static_names) - set(current_names)
-
-    # Extra files/directories
-    extra_names_tmp = set(current_names) - set(static_names)
-    check_extra_names = []
-    for extra_name in extra_names_tmp:
-        if extra_name in dynamic_names:
-            check_extra_names.append(extra_name)
-        else:
-            extra_names.append(extra_name)
-
-    # Different files/directories
-    different_items = {}
-    # Static
-    for item in static_items:
-        if item not in missing_names and static_items[item] != current_items[item]:
-            different_names.append(item)
-            different_items[item] = static_items[item]
-    # Dynamic
-    for check_extra_name in check_extra_names:
-        if dynamic_items[check_extra_name] != current_items[check_extra_name]:
-            different_names.append(check_extra_name)
-            different_items[check_extra_name] = dynamic_items[check_extra_name]
-
-    assert not different_names
-
-
-# The test receives the environment with wazuh-agent installed and started.
+import pytest
+import re
+import json
+
+from ..helpers import constants, utils
+from ..helpers.installer import WazuhInstaller
+from ..helpers.checkfiles import CheckFile
+from ..helpers.hostinformation import HostInformation
+
+@pytest.fixture
```

Comment:
```suggestion

@pytest.fixture
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/4998/commits/db29c08420c01890d8ca7fedddeb25b94ab07fc3